manifests/params.pp

```diff
@@ -184,6 +183,12 @@
   $server_http_port           = 8139
   $server_http_allow          = []
 
+  # use puppetserver (JVM) or puppet master (Ruby)?
+  $server_implementation = $aio_package ? {
+    true    => 'puppetserver',
+    default => 'master',
+  }
+
```

Comment:
I'm strongly against that `globals` pattern, it's making things for non-hiera setups much worse in some aspects, especially when using something like foreman-installer, see #361. `aio_package` in theforeman-puppet is also not overrideable and I think duplicating the pattern here is no problem unless somebody (probably me) starts implementing support for puppetserver on platforms without AIO (e.g. FreeBSD). `server_implementation` is overrideable anyway.